packages/labs/test/esbuild/BUILD.bazel

```diff
@@ -0,0 +1,59 @@
+load("//packages/jasmine:index.bzl", "jasmine_node_test")
```

Comment:
This is a great initial test. If possible, making an integration test within the `examples` directory will test that the actual npm packaging logic is correct. Not required for labs, but it gives me some piece of mind when I make one